element_array_ephys/ephys_acute.py

```diff
@@ -757,3 +983,39 @@ def generate_electrode_config(probe_type: str, electrodes: list):
 
     return electrode_config_key
 
+
+def get_recording_channels_details(ephys_recording_key):
+    channels_details = {}
+
+    acq_software, sample_rate = (EphysRecording & ephys_recording_key).fetch1('acq_software',
+                                                              'sampling_rate')
+
+    probe_type = (ProbeInsertion * probe.Probe & ephys_recording_key).fetch1('probe_type')
+    channels_details['probe_type'] = {'neuropixels 1.0 - 3A': '3A',
+                                      'neuropixels 1.0 - 3B': 'NP1',
+                                      'neuropixels UHD': 'NP1100',
+                                      'neuropixels 2.0 - SS': 'NP21',
+                                      'neuropixels 2.0 - MS': 'NP24'}[probe_type]
+
+    electrode_config_key = (probe.ElectrodeConfig * EphysRecording & ephys_recording_key).fetch1('KEY')
+    channels_details['channel_ind'], channels_details['x_coords'], channels_details[
+        'y_coords'], channels_details['shank_ind'] = (
+            probe.ElectrodeConfig.Electrode * probe.ProbeType.Electrode
+            & electrode_config_key).fetch('electrode', 'x_coord', 'y_coord', 'shank')
+    channels_details['sample_rate'] = sample_rate
+    channels_details['num_channels'] = len(channels_details['channel_ind'])
+
+    if acq_software == 'SpikeGLX':
+        spikeglx_meta_filepath = get_spikeglx_meta_filepath(ephys_recording_key)
+        spikeglx_recording = spikeglx.SpikeGLX(spikeglx_meta_filepath.parent)
+        channels_details['uVPerBit'] = spikeglx_recording.get_channel_bit_volts('ap')[0]
```

Comment:
Since we are using the value for the first channel, is it the same for all channels?

Reply:
Looks like it is same for all channels ([source](https://github.com/ttngu207/element-array-ephys/blob/c2f8aeaa7aad48c602484b9358a569d2b54d69c4/element_array_ephys/readers/spikeglx.py#L106-L111)).